widget/app/src/constants.ts

```diff
@@ -1 +1,5 @@
 export const WC_PROJECT_ID = 'e24844c5deb5193c1c14840a7af6a40b';
+export const TREZOR_MANIFEST = {
+  appUrl: 'https://rango.exchange/',
```

Comment:
```suggestion
  appUrl: 'https://widget.rango.exchange/',
```

Reply:
Shouldn't we put the correct email address and URL for the app?

Reply:
fixed